fleet/lib/kube-prometheus-stack/fleet.yaml

```diff
@@ -136,3 +136,21 @@ targetCustomizations:
         prometheus:
           prometheusSpec:
             remoteWrite: *tu-remotewrite
+  - name: local-low-prometheus
+    clusterSelector:
+      matchExpressions:
+        - key: management.cattle.io/cluster-display-name
+          operator: In
+          values:
+            - local
```

Comment:
This target will never match because there is already an explicit name match on line 83: https://github.com/lsst-it/k8s-cookbook/pull/906/files#diff-d143cb3f98049980638fd7a43dc711a0f0994c6b661e879d9215d9a663329322R83 

and then all of the dev/ls/cp clusters will match https://github.com/lsst-it/k8s-cookbook/pull/906/files#diff-d143cb3f98049980638fd7a43dc711a0f0994c6b661e879d9215d9a663329322R83

Reply:
i needed more knowledge, so now i know that multiple `matchExpressions` act like  `AND` and must match all the `Keys` in the selector. 💪🏻